src/zzz/zh/docs/feat_world_patrol.md

```
@@ -0,0 +1,39 @@
+- 在使用前，请务必运行一次位于 `游戏助手` 内的 `鼠标校准` 
+:::
+
+## 快速开始-锄大地
+
+1. 在 `游戏助手` 内的 `鼠标校准` 页面进行鼠标校准
+2. 在 `锄大地` 页面选择路线
+3. 启动 `锄大地` 功能
+
+## 锄大地
+
+锄大地主体由[自动战斗](./feat_battle_assistant.md)和[路线]()两部分组成
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)